src/main/java/soonwee/duke/Duke.java

```diff
@@ -0,0 +1,160 @@
+package soonwee.duke;
+
+import java.io.IOException;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+import java.time.LocalDateTime;
+
+/**
+ * Represents a Duke instance. A Duke instance will prompt user which bot they
+ * want to use (1. Echo Bot, 2. Tasks List Bot).
+ */
+public class Duke {
+
+    public static final int TODO = 1;
+    public static final int DEADLINE = 2;
+    public static final int EVENT = 3;
+    public static final int FIND = 4;
```

Comment:
Great practice of capitalising the constant variables. Just curious but is there a reason why you chose to use constant variables over enumerations?

Reply:
Same question

Reply:
Yes I totally agree with this. I have created an enum class to resolve this. Thanks :)